scrapers/GIGA.yml

```
@@ -105,7 +105,7 @@ xPathScrapers:
       Director:
         selector: //*[@id="works_txt"]/ul/li[5]/dl/dd
       Image:  
-        selector: //*[@id="works_pic"]/ul/li[1]/img/@src
+        selector: //*[@id="works_pic"]/ul/li[1]/a/@href
```

We can use both selectors here and Stash will return the first one that gives a result 🙂

```suggestion
        selector: //*[@id="works_pic"]/ul/li[1]/a/@href | //*[@id="works_pic"]/ul/li[1]/img/@src
```

---

Commited change 👍